app/assets/javascripts/admin/modules/taxonomy_tree_checkboxes.js

```diff
@@ -56,8 +81,10 @@
         and the content could be shown to users seeking any of them.
         */
         if (checked) {
+          checkboxTrackClick("checkboxClickedOn", options);
```

Comment:
When we automatically check/uncheck the ancestors/descendants, does this trigger another event that causes those checkboxes to be tracked as click?

I think we'd want to track just the user behaviour, since we are using number of clicks as a proxy for how confident the users are when using the interface, and the actual number of checked/unchecked depends on where in the taxonomy you're tagging to.